### PR TITLE
Improved: slider

### PR DESCRIPTION
**Pull request by math-GH**
After:
grafik

1. Fix slider width (750px). (There is no benefit to have a wider slider.
2. Inputs/dropdowns/textareas could have a width of 50% and 100%. It improves the general layout and give the space for a better text input. The "extend" inputs are replaced with 100% widths inputs. No expanding & jumping inputs anymore.
3. slim scrollbar for slider content
4. deleted: the feed description. It is editable some centimeters below. It does not make sense to display redundant information.

In mobile view it looks also much better
grafik

Changes proposed in this pull request:

- templates (.phtml)
- CSS

How to test the feature manually:

1. go to subscription management
2. edit a feed
3. see the slider content

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

File: app/views/helpers/feed/update.phtml [hunk 1]
```diff
@@ -8,8 +8,6 @@
 		<a href="<?= _url('stats', 'repartition', 'id', $this->feed->id()) ?>"><?= _i('stats') ?> <?= _t('sub.feed.stats') ?></a>
 	</div>
 
-	<p><?= sanitizeHTML($this->feed->description()) ?></p>
```
Comment (Alkarex) [inline app/views/helpers/feed/update.phtml:11]: This was not supposed to be removed. https://github.com/FreshRSS/FreshRSS/issues/5320

Comment (math-GH) [reply app/views/helpers/feed/update.phtml:11]: It was supposed to. See number 4:

> deleted: the feed description. It is editable some centimeters below. It does not make sense to display redundant information.

Comment (Frenzie) [reply app/views/helpers/feed/update.phtml:11]: It's not redundant fwiw; the one is plain text input and the removed one is HTML.

Comment (math-GH) [reply app/views/helpers/feed/update.phtml:11]: Here a screen shot of FreshRSS 1.19.0.
What is the benefit to see the description in (1), while it is written in (2) too?

grafik

Comment (Alkarex) [reply app/views/helpers/feed/update.phtml:11]: Just for the record here as well: the (1) is in HTML, meaning that it can contain hyperlinks, images, etc.

**Pull request merged**